.gitignore

```diff
@@ -2,6 +2,7 @@
 .idea
 .mvn
 build
+doc-artifacts
```

Comment:
interesting, do we still have places were we generate `doc-artifacts` outside of `target` dir?

Reply:
@mattmetlis do you know if anything else is pending to merge this?